_manual/pull_request.md

```diff
@@ -5,6 +5,10 @@ manual: cont
 order: 30
 ---
 
-_...to be written..._
+## Some rules
 
-Isn't a good way for you to **start [contributing documentation](/manual/writing_documentation)?**
+- Please do not put too much into one PR. To ease reviewing and testing, it is prefered that a pull request handles only one single problem. That because one obstacle/problem and it delays the possible good stuff to merge.
+- Please give pull requests a good summary, `$name fixes` as an example, to make the review process easier.
```

Comment:
Oh, sorry, you misunderstood. That was a bit ironic reference for your 'Trebmuh fixes' PR title..

Reply:
Still a good advice, isn't?